pom.xml

```diff
@@ -0,0 +1,101 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>org.jenkins-ci.plugins</groupId>
+        <artifactId>plugin</artifactId>
+        <version>4.66</version>
+        <relativePath />
+    </parent>
+
+    <artifactId>scm-filter-jervis</artifactId>
+    <version>${revision}-${changelist}</version>
+    <packaging>hpi</packaging>
+
+    <name>SCM Filter Jervis Plugin</name>
+    <url>https://github.com/jenkinsci/scm-filter-jervis-plugin</url>
+    <licenses>
+        <license>
+            <name>MIT License</name>
+            <url>http://opensource.org/licenses/MIT</url>
+        </license>
+    </licenses>
+
+    <developers>
+        <developer>
+            <id>sag47</id>
+            <name>Sam Gleske</name>
+        </developer>
+    </developers>
```

Comment:
no need for this, plugin site uses repository permissions updater data now as it's more accurate

Reply:
Ok I will remove it.  Thanks!

Reply:
I need to check that it compiles the groovy after sleeping on it

Reply:
Removed but the plugin is broken per my latest review.